liquibase-standard/src/main/java/liquibase/hub/core/StandardHubService.java

```diff
@@ -357,12 +357,12 @@ public String shortenLink(String url) throws LiquibaseException {
     }
 
     /**
-     * Query for a changelog ID.  If no result we return null
-     * We cache this result and a map
+     * Queries for a changelog ID. If no result is found, it returns {@code null}.
```

Comment:
This whole class is going away with Hub decommission. I bet we see that show up soon now that the Hub decommission branch is merged to master ...

Reply:
I updated this branch with master and this file is gone. I'm also removing the `StandardHubServiceTest` class as well.

Reply:
Thanks!